### ISSUE TITLE
Decompiler can emit invalid Hero annotations

### ISSUE THREAD

**Issue by Hawkbat** [Zezombye/overpy#70]
Don't have time to make a minimum reproducible example, but I tried decompiling this workshop script:  https://gist.github.com/Hawkbat/fea2fefbb1208dd200e24848dc4ab098

And it produced this decompiled script: https://gist.github.com/Hawkbat/613e9b9c7ce4b495e7ad7fb13d0fa8ca

The decompiled output contains Hero annotations that look like this:
```
@Rule "Spawn Downed Dummy"
@Event eachPlayer
@Team 1
@Hero 
if eventPlayer.isCrouching():
    createDummy(random.choice(eventPlayer.getAllowedHeroes()), Color.TEAM_1, -1, nearestWalkablePosition(eventPlayer.getPosition()), Vector.DOWN)
    wait(2)
    damage(getLastCreatedEntity(), null, 99999)
```

This causes the extension to error out with "Error: Cannot read property 'text' of undefined" instead of compiling the file. Removing the invalid annotations allows the script to be copied to clipboard successfully.

**Comment by Hawkbat** [comment]
Might be related to `@Slot` annotations? All of the rules with invalid `@Hero` annotations have Slot conditions but not Hero conditions.

**Comment by Zezombye** [comment]
Seems like it, thanks for reporting!

Will take a look tomorrow.

**Comment by Zezombye** [comment]
Fixed in 2.0.3.
